etc/nginx/vhosts.d/openqa-locations.inc

```diff
@@ -56,4 +56,5 @@ location / {
     proxy_set_header X-Forwarded-Server $host;
     proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
     proxy_set_header X-Forwarded-Proto $scheme;
+    rewrite ^/tests/.*/asset/.*/[wW]in.*$ https://www.microsoft.com/software-download/windows10ISO last;
```

Comment:
Wasn't that asset**s** in the original rule?

Reply:
See my comment in the ticket. The `/assets` url was commented out, so I didn't port that one.

Reply:
Makes sense, I was just looking at the original ticket description.